assets/js/permutive.js

```diff
@@ -1,6 +1,22 @@
-const contentId = document.documentElement.dataset.contentId;
-const Permutive = require('alphaville-ui')['permutive'];
+const nPermutive = require('n-permutive');
 
-Permutive.initPermutive();
-Permutive.setUserAndContent(contentId);
-Permutive.setPermutiveSegments();
+const PERMUTIVE_CREDENTIALS = {
+	publicId: 'e1c3fd73-dd41-4abd-b80b-4278d52bf7aa',
```

Comment:
Are we not putting these into environment variables, because they are already public?

Reply:
Yes, they are public credentials. They were previously set up in `alphaville-ui` using o-permutive origami package. Moved the set up here as now we are using the n-permutive npm package.

Reply:
Same approach used for `alphaville-longroom` in the other PR https://github.com/Financial-Times/alphaville-longroom/pull/65